content/workshops/node/_index.md

```diff
@@ -1,11 +1,6 @@
 ---
-title: Node 
+title: Node
+ready: true
 ---
 
-## Introduction (slides)
-
-- [Introduction to Node](https://docs.google.com/presentation/d/1dtrsWOWjoA9A9nTmznCkdYz0oRK6O76jXRINp0tzjrQ/edit?usp=sharing)
-
-- [Asynchronous vs Synchronous](https://docs.google.com/presentation/d/19ppsjmkKAvJcasK_BnjptTUfF4KQwaInDKVkhK1b6vA/edit?usp=sharing)
-
-## Challenges 
\ No newline at end of file
+- Resources related to this topic can be found here {{% contentlink "topics/Node" %}}
```

Comment:
file naming conventions should be followed. use a small letter for node, otherwise the links wont work.

As a programmer working on an existing code base you should always follow the conventions that are in place.